_observing-your-data/alerting/index.md

```diff
@@ -23,34 +21,35 @@ The following table lists alerting terminology commonly used in OpenSearch and t
 Term | Definition
 :--- | :---
 Monitor | Job that runs on a defined schedule and queries OpenSearch indexes. The results of these queries are then used as input for one or more triggers.
-Trigger | Conditions that, if met, generate alerts. See [Triggers]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/triggers/).
+Trigger | Conditions that, if met, generate alerts.
 Alert | Event associated with a trigger. When an alert is created, the trigger performs actions, including sending notifications.
-Action | Specific task that is performed when an alert is triggered. See [Actions]({{site.url}}{{site.baseurl}}/observing-your-data/alerting/actions/).
-Notification | Message that is sent to users when an alert is triggered. See [Notifications]({{site.url}}{{site.baseurl}}/notifications-plugin/index/).
+Action | Specific task that is performed when an alert is triggered.
+Notification | Message that is sent to users when an alert is triggered.
 
-## Alert states
+## Alerting settings
+
+Some key alerting settings in OpenSearch include the ability to customize trigger conditions, severity levels, notification channels, and recurrence intervals for monitors. You can modify settings like notification message templates, throttling to avoid duplicate alerts, and status expiration periods. 
+
+<SME: Do we have specific settings to include in this section? If so, please provide the setting name, description, and default setting.>
```

Comment:
SME: Do we have specific settings to include in this section? If so, please provide the setting name, description, and default setting, or link me to the source where I can gather this info.

Reply:
@AWSHurneyt , can you share details with @vagimeli

Reply:
@vagimeli 
Here's the documentation that discusses the items mentioned in that paragraph in more detail.
This page describes trigger conditions, and severity levels.
https://opensearch.org/docs/latest/observing-your-data/alerting/triggers/#trigger-variables

Notification plugin channels are associated with alerting monitor by adding "actions" to the monitor triggers. This page describes adding actions, using mustache template to create the notification messages, and throttling.
https://opensearch.org/docs/latest/observing-your-data/alerting/actions/

@lezzago do you know of documentation for the `status expiration periods` mentioned in the paragraph?

Reply:
Revised.